packages/crypto/src/proof.ts

```diff
@@ -0,0 +1,254 @@
+import { uint8ArrayFromHexString } from "@turnkey/encoding";
+import type { v1AppProof, v1BootProof } from "@turnkey/sdk-types";
+import { p256 } from "@noble/curves/p256";
+import { sha256 } from "@noble/hashes/sha2";
+import * as CBOR from "cbor-js";
+import * as x509 from "@peculiar/x509";
+import { Crypto as PeculiarCrypto } from "@peculiar/webcrypto";
+import { createHash, createPublicKey, createVerify } from "node:crypto";
+import { AWS_ROOT_CERT_PEM, AWS_ROOT_CERT_SHA256 } from "./constants";
+
+x509.cryptoProvider.set(new PeculiarCrypto());
+
+// verify goes through the following verification steps for an app proof & boot proof pair:
+//  - Verify app proof signature
```

Comment:
(nit) I think the standard for function-level JS docs is:
```ts
/**
 * bla bla bla
 * ...
 */
```

https://jsdoc.app/about-getting-started